docs/docs/how-to/add_users_to_project.md

```diff
@@ -8,8 +8,52 @@ sidebar_position: 100
 * The `minder` CLI application
 * A Stacklok account
 
-## Roles Overview
-When incorporating a user into your project, it's crucial to assign them the appropriate role based on their responsibilities and required access levels. 
+## Overview
+
+To add a new user to your project, you need to:
+
+1. Identify your project ID
+2. Create a new user account
```

Comment:
This is something that the other user needs to do, right?

Maybe

```suggestion
2. Have the user log in to Minder
```

Reply:
Good call out, thanks. I updated the header below to match.